deploy/crds/chaosexperiment_crd.yaml

```diff
@@ -58,20 +58,30 @@ spec:
                     name: 
                       type: string
                 permissions: 
-                  type: object
-                  properties: 
-                    apiGroups: 
-                      type: array
-                      items: 
-                        type: string
-                    resources: 
-                      type: array
-                      items: 
-                        type: string
-                    verbs: 
-                      type: array
-                      items: 
-                        type: string
+                  type: array
+                  items:
+                    type: object
```

Comment:
For each object, let us define the min properties (3) & the required items (apigroup/resource/verbs) as a set!